eZ/Publish/API/Repository/Tests/ContentTypeServiceTest.php

```diff
@@ -1237,6 +1237,49 @@ public function testLoadContentTypeDraftThrowsNotFoundException()
         /* END: Use Case */
     }
 
+    /**
+     * Test for the loadContentTypeDraft() method.
+     *
+     * @see \eZ\Publish\API\Repository\ContentTypeService::loadContentTypeDraft()
+     * @expectedException \eZ\Publish\API\Repository\Exceptions\NotFoundException
```

Comment:
Note: `expectedException` annotation will be deprecated, so either now or on merge up into `master` it needs to be changed to `expectException` method call. See [8.0.0-beta1 version of ContentTypeServiceTest](https://github.com/ezsystems/ezpublish-kernel/blob/v8.0.0-beta1/eZ/Publish/API/Repository/Tests/ContentTypeServiceTest.php#L1237).